retrieval/scrape_test.go

```diff
@@ -222,54 +222,42 @@ func TestScrapePoolReload(t *testing.T) {
 	}
 }
 
-func TestScrapePoolReportAppender(t *testing.T) {
+func TestScrapeLoopWrapSampleAppender(t *testing.T) {
 	cfg := &config.ScrapeConfig{
 		MetricRelabelConfigs: []*config.RelabelConfig{
-			{}, {}, {},
+			{
+				Action:       config.RelabelDrop,
+				SourceLabels: model.LabelNames{"__name__"},
+				Regex:        config.MustNewRegexp("a.*"),
+			},
+			{
+				Action:       config.RelabelDrop,
```

Comment:
Given there's an actual test below using data similar to this, regexes of foo and bar would probably be a tiny bit clearer.